forge-gui/res/cardsfolder/t/two_streams_facility.txt

```diff
@@ -0,0 +1,17 @@
+Name:Two Streams Facility
+ManaCost:no cost
+Types:Plane Apalapucia
+T:Mode$ PlaneswalkedTo | ValidCard$ Card.Self | Execute$ TrigChoose | TriggerDescription$ Whenever you planeswalk to CARDNAME and at the beginning of the first upkeep of the game, each player chooses green anchor or red waterfall.
+T:Mode$ Phase | FirstUpkeepThisGame$ True | Execute$ TrigChoose | TriggerZones$ Command | Secondary$ True | TriggerDescription$ Whenever you planeswalk to CARDNAME and at the beginning of the first upkeep of the game, each player chooses green anchor or red waterfall.
+SVar:TrigChoose:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ DBChoice | ClearRememberedBeforeLoop$ True
+SVar:DBChoice:DB$ GenericChoice | Defined$ Remembered | Choices$ GreenAnchor,RedWaterfall | ShowChoice$ True
```

Comment:
what would happen if you do `DB$ GenericChoice | Defined$ Player | Choices$ GreenAnchor,RedWaterfall` and make the Loop Player available inside the SubAbility via ChooseGenericEffect ?

Reply:
Sounds good

Reply:
i don't know if we can do this for all Cards or if that would break some cards

Reply:
I think leaving it as a Param gives us the flexibility to more easily tweak it later... and only break cards we add the param to 😬 

I refactored some more cards... I am scared of messing with "Eureka" or "Hypergenesis"

Reply:
we could always update them later